lib/subscriptions.ts

```diff
@@ -143,66 +239,28 @@ export const evaluate = async ({
 				return;
 			}
 
-			const creatorSession = await getSession(creatorId);
-
-			if (!creatorSession) {
+			// User already got notification, because he was mentioned
+			if (getMentions(message).includes(creator.slug)) {
 				return;
 			}
```

Comment:
Maybe we should just try to insert another notification and ignore duplicate contract error.

Reply:
I think just insert another notification, I don't think you even need to worry about using a deterministic slug. It's not a huge issue to get notified again if the contract changes I think 🤔

Reply:
Or just ignore the duplicate error for sure!